com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -349,7 +381,12 @@ UniversalRenderPipelineEditorResources editorResources
             }
         }
 #endif
-
+        /// <summary>
+        /// Used to create a renderer give a <c>RendererType</c>.
```

Comment:
```suggestion
        /// Used to create a renderer given a <c>RendererType</c>.
```

Reply:
This seems to not create the Renderer but to initialize the RendererData that is required by the renderer.

Reply:
Updated it to <summary>
        /// Used to initialized the rendererData that is required by the renderer.
        /// </summary>
        /// <param name="type">The <c>RendererType</c> of the new renderer it should initialized within this asset.</param>
        /// <returns></returns>
        /// <see cref="RendererType"/>